dev/TEAL.md

```diff
@@ -634,8 +636,10 @@ different transaction types, are rejected by `itxn_submit`.
 | `itxn_submit` | execute the current inner transaction group. Fail if executing this group would exceed the inner transaction limit, or if any transaction in the group fails. |
 | `itxn f` | field F of the last inner transaction |
 | `itxna f i` | Ith value of the array field F of the last inner transaction |
+| `itxnas f` | Ath value of the array field F of the last inner transaction |
```

Comment:
Since `itxna` is essentially deprecated, do we really want to add `itxnas`?

Reply:
I went for consistency over opcode savings.  I think we need to think about deprecation soon, to recover some opcode space.  The itxn* opcodes might go, `balance` and `min_balance` might go (because `acct_params_get` exists now). `gaid*` might go (because `txn Created*Id` exists) Hashing and verifying might get grouped into one opcode with a field (if we enhance the cost model to allow differences based on field).

I would prefer to do that all at once, rather than have the missing opcode now.